### PR TITLE
Fix bugs #432 for fans valerii 

### PR DESCRIPTION
**Pull request by ValDekh**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue
[Admin/Sources block] An admin can add more than 4000 symbols to the text field #432 

Chage text limit from 1000 to 10000 in Text Content field for fan

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: Streetcode/StreetCode.Client
Comment (Tysyatsky) [inline Streetcode/StreetCode.Client:1]: why is it here?

File: Streetcode/Streetcode.WebApi/appsettings.Local.json
Comment (Tysyatsky) [inline Streetcode/Streetcode.WebApi/appsettings.Local.json:1]: change it back to normal

File: Streetcode/Streetcode.WebApi/appsettings.json
Comment (Tysyatsky) [inline Streetcode/Streetcode.WebApi/appsettings.json:1]: and here

File: Streetcode/Streetcode.DAL/Persistence/ScriptsMigration/DDL_last_mig.sql
Comment (Tysyatsky) [inline Streetcode/Streetcode.DAL/Persistence/ScriptsMigration/DDL_last_mig.sql:1]: did you add new changes from migration to the script and do you still need migration files that was generated?

**Review by Tysyatsky — CHANGES_REQUESTED** [review]

File: Streetcode/Streetcode.WebApi/appsettings.Local.json
Comment (ValDekh) [reply Streetcode/Streetcode.WebApi/appsettings.Local.json:1]: done

File: Streetcode/Streetcode.WebApi/appsettings.json
Comment (ValDekh) [reply Streetcode/Streetcode.WebApi/appsettings.json:1]: done

File: Streetcode/Streetcode.DAL/Persistence/ScriptsMigration/DDL_last_mig.sql
Comment (ValDekh) [reply Streetcode/Streetcode.DAL/Persistence/ScriptsMigration/DDL_last_mig.sql:1]: Yes, I thing we need both

File: Streetcode/StreetCode.Client
Comment (ValDekh) [reply Streetcode/StreetCode.Client:1]: done

**Pull request merged**